strapi/strapi
# Adding icons for Single Types and Collection type

### What does it do?

Adds an icon for ST and CT when editing a collection type.

<img width="973" alt="Screenshot 2021-02-16 at 09 35 52" src="https://user-images.githubusercontent.com/3874873/108038129-c6c2c780-703a-11eb-8a85-c93450208ad4.png">


### How to test

- Log in
- Select Content Type Builders on the left
- Select a Content Type on the sub left side bar
- Click the little pen close to the Content Type name
- You should see the radio buttons with the icons
